docs/reference/service/configuration.md

```diff
@@ -269,6 +270,18 @@ Note that OTLP traces can be consumed by different solutions, like [Jaeger](http
   }
   ```
 
+
+### `clients`
+
+An array of [Platformatic Client](/reference/client/introduction.md) configurations that will be loaded by Platformatic Service.
+
+- **`serviceId`** (`string`) - The ID of Platformatic Service inside the Platformatic Runtime. Used only in Platformatic Runtime context.
```

Comment:
can you make "runtime context" a link pointing to some definition of that (add it if it's missing).